src/chord.py

```diff
@@ -0,0 +1,101 @@
+from os import O_TRUNC
+import parselmouth
+import math
+
+
+class Sound:
+    def __init__(self, note, timestamp=0, duration=0):
+        self.note = note
+        self.timestamp = timestamp
+        self.duration = duration
+
+    @property
+    def symbol(self):
+        if self.note == None:
+            return 'None'
+        #           0    1     2    3     4
+        symbols = ['C', 'C#', 'D', 'D#', 'E',
+                   # 5   6     7    8     9    10    11
+                   'F', 'F#', 'G', 'G#', 'A', 'A#', 'H']
+        return symbols[self.note]
+
+    @property
+    def note(self):
+        return self._note
+
+    @note.setter
+    def note(self, note):
+        self._note = note % 12 if note else note
+
+    def __str__(self):
+        return f"{self.timestamp}: {self.symbol}"
+
+    def __eq__(self, other):
+        return self.note == other.note
+
+    __repr__ = __str__
```

Comment:
Remove this and make use of the Sound class from my commits you accidentally included in PR.